extensions/terminal-suggest/src/fig/figInterface.ts

```diff
@@ -52,11 +52,10 @@ export async function getFigSuggestions(
 		if (!specLabels) {
 			continue;
 		}
-
 		for (const specLabel of specLabels) {
 			const availableCommand = (osIsWindows()
 				? availableCommands.find(command => (typeof command.label === 'string' ? command.label : command.label.label).match(new RegExp(`${specLabel}(\\.[^ ]+)?$`)))
-				: availableCommands.find(command => (typeof command.label === 'string' ? command.label : command.label.label).startsWith(specLabel)));
+				: availableCommands.find(command => (typeof command.label === 'string' ? command.label : command.label.label) === (specLabel)));
```

Comment:
Does this still work properly for aliases like `alias lss=ls`?

Reply:
Good point, think we also want to factor in `command.definitionCommand` here to address that

Reply:
tested and that works too with the most recent commit